packages/ds-react-core/src/ui/index.ts

```diff
@@ -1,3 +1,3 @@
-export { Button, type ButtonProps } from "./Button/index.js";
-export { Header, type HeaderProps } from "./Header/index.js";
-export { Page } from "./Page/index.js";
+export * from "./Button/index.js";
+export * from "./Header/index.js";
+export * from "./Page/index.js";
```

Comment:
These index files already export named and type exports, we can use `export *` for simplicity